apps/cowswap-frontend/src/legacy/state/orders/actions.ts

```diff
@@ -136,6 +136,7 @@ export interface AddPendingOrderParams {
   id: UID
   chainId: ChainId
   order: SerializedOrder
+  isSafeWallet: boolean
```

Comment:
`isSafeWallet` is relevant later to give a full context on the order. This way, the middleware have everything they need to show the toast messages (or notify about the messages through the event emitter) without relying on hooks or any other dark magic. The message will be self contained